README.md

```diff
@@ -70,6 +70,25 @@ If you like details and wish to know about all the AT interactions between the m
         },
 ```
 
+### Turning on the tracing and trace level
+
+If you like to add more traces or follow the current ones you can turn traces on by changing `mbed-trace.enable` in mbed_app.json
+
+```"target_overrides": {
+        "*": {
+            "target.features_add": ["LWIP", "COMMON_PAL"],
+            "mbed-trace.enable": false,
+```
+
+Changing the trace level is done also in mbed_add.json by changing value in `trace-level`
```

Comment:
Maybe something like... After you have defined 'mbed-trace.enable: true', you can set trace levels...